research/conditional/utils/conditional_trainer.py

```diff
@@ -336,6 +337,16 @@ def _log_train_stats(self, loss_value, step):
                     value=stats.acc / stats.interval,
                     iteration=step,
                 )
+                self.logger.report_scalar(
+                    title=f"{name}_flops",
+                    value=stats.acc / stats.interval,
+                    iteration=step * self.flops_per_step,
+                )
+                self.logger.report_scalar(
+                    title=f"{name}_tokens",
+                    value=stats.acc / stats.interval,
+                    iteration=step * self.batch_size * self.max_sequence_length,
+                )
```

Comment:
Are you sure this is correct? Shouldn't 'iteration' be a step? Like this?
```suggestion
                self.logger.report_scalar(
                    title=f"{name}_flops",
                    value=step * self.flops_per_step,
                    iteration=step,
                )
                self.logger.report_scalar(
                    title=f"{name}_tokens",
                    value=step * self.batch_size * self.max_sequence_length
                    iteration=step,
                )
```
I've got the point to measure how much we actually processed achieving given loss, but not sure it is possible this way

Reply:
I think that's correct and does exactly what we wanted to: produce a loss/tokens or a loss/flops plot.